src/eggd_sompy.sh

```
@@ -94,8 +125,10 @@ main() {
             dx-upload-all-outputs --parallel
         else
             echo "Query VCF and query VCF filename inputs are NOT the same"
+            exit -1
         fi
     else
         echo "Either Query VCF and query VCF filename inputs are missing"
+        exit -1
```

Shellcheck (0.11.0)

[error] Can only exit with status 0-255. Other data should be written to stdout/stderr.

(SC2242)